docs/zq/README.md

```diff
@@ -216,7 +215,7 @@ we felt that the design of having a uniform default had worse consequences:
 and deploy to production systems that were accidentally using ZSON instead of
 the much more efficient ZNG format because the `-f zng` had been mistakenly
 omitted from some command.  The beauty of Zed is that all of this "just works"
-but it would otherwise perform horribly.
+but it would otherwise perform poorly.
```

Comment:
I guess this is just style, but from a self-esteem perspective, I'd argue that nothing in Zed performs _horribly_. :wink: